vllm/distributed/device_communicators/tpu_communicator.py

```diff
@@ -0,0 +1,35 @@
+from typing import Union
+
+import torch
+from torch.distributed import ProcessGroup
+
+from vllm.platforms import current_platform
+
+if current_platform.is_tpu():
+    import torch_xla.core.xla_model as xm
+    from torch_xla._internal import pjrt
+
+
+class TpuCommunicator:
+
+    def __init__(
+        self,
+        group: ProcessGroup,
+        local_rank: int,
+        world_size: int,
+    ):
+        del group  # Unused.
+        if not current_platform.is_tpu():
+            self.disabled = True
+            return
+        self.disabled = False
+
+        pjrt.initialize_multiprocess(local_rank, world_size)
```

Comment:
you can get rank and world size from the `group`

Reply:
Thanks for letting me know! Updated the PR. PTAL.

Reply:
do you need to remove the code inside tpu worker? I don't know if pjrt and xm support initialization for multiple times.

Reply:
@youkaichao Good point. That's actually updated in #5871. In the current main branch, there's no code initializing the XLA's distributed runtime.